processing/data_collection/gazette/schema.json

```diff
@@ -0,0 +1,58 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "date": {
+            "description": "Date of the gazzete",
+            "type": "string"
```

Comment:
**issue**: Gazette date is a well defined string with YYYY-MM-DD. It would be a good idea to match this pattern when checking this field.

Reply:
that's a good idea, I thought the date were a "datetime" object